department-of-veterans-affairs/abd-vro
# Add a svc-bip-integration-test to the build steps in github actions

## What was the problem?


Since we are making changes to the svc-bip-api having an integration test is helpful for ensuring we are pushing code that does not cause breaking changes to the api.

Associated tickets or Slack threads:
- #2133

## How does this fix it?[^1]


Adds the integration-test-.yml to github actions for adding to the test-code build step

## How to test this PR
- Run the workflow in github and see if it passes.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
